controller/src/beerocks/master/tasks/optimal_path_task.cpp

```diff
@@ -362,9 +362,11 @@ void optimal_path_task::work()
          * There is a high probability to get responsiveness less than 100% and therefore, the
          * threshold was changed to 50% as part of demo optimizations.
          * This hardcoded threshold is temporary and shall be revised in the future. 
+         * update: increasing to 80% since in bandsteering scenario there are only 2 bands
+         * and 50% means optimal path always will fail to find an additional candidate.
          */
-        if (responsiveness_precentage_11k < 50) {
-            TASK_LOG(DEBUG) << "11k measurement request responsiveness is less than 50%";
+        if (responsiveness_precentage_11k < 80) {
```

Comment:
Seeing that you change this now, it may be better to add it to the conf file or even UCI.

Reply:
or at least a define

Reply:
please use `constexpr` (even if at a later stage you move this definition to the config file.